extensions/panache/panache-hibernate-common/deployment/src/main/java/io/quarkus/panache/common/deployment/PanacheHibernateCommonResourceProcessor.java

```diff
@@ -126,7 +126,7 @@ void replaceFieldAccesses(CombinedIndexBuildItem index,
                 }
                 produced.add(cn);
                 transformers.produce(
-                        new BytecodeTransformerBuildItem(cn, panacheFieldAccessEnhancer, entityClassNamesInternal));
+                        new BytecodeTransformerBuildItem(true, cn, panacheFieldAccessEnhancer, entityClassNamesInternal, true));
```

Comment:
FWIW the first `true` is making the transformer eager, whereas it previously was lazy.

Not sure this changes anything though... It seems this information is not used anywhere. It cascades down to this code, which writes to the set but never reads from it :x 

https://github.com/quarkusio/quarkus/blob/3374799f6d0523767d978ecbd35dbbe96f258ad5/core/deployment/src/main/java/io/quarkus/runner/bootstrap/StartupActionImpl.java#L63-L64

Reply:
As I mentioned, we're not keeping them lazy. The boolean is ultimately used to populate a `Set` which nobody ever reads from.

Reply:
ho... fun :)